dags/solutions_team/solutionsteam_tf_nightly_supported.py

```diff
@@ -112,7 +112,7 @@
       runtime_version=RuntimeVersion.TPU_VM_TF_NIGHTLY_POD.value,
   )
 
-  embedding_dim = 32
```

Comment:
is this intended to be part of this PR?

Reply:
this was from upstream, I just merged with master so this change is now gone.